app/src/main/java/com/example/pygmyhippo/user/ProfileFragment.java

```diff
@@ -60,6 +72,39 @@ public void onAttach(@NonNull Context context) {
         }
     }
 
+    /**
+     * Registers a photo picker activity launcher in single-select mode and sets the profile image to the new URI
+     * @author Jennifer
+     * @version 1.0
+     */
+    ActivityResultLauncher<PickVisualMediaRequest> pickMedia =
+            registerForActivityResult(new ActivityResultContracts.PickVisualMedia(), uri -> {
+                // Callback is invoked after the user selects a media item or closes the
+                // photo picker.
+                if (uri != null) {
+                    imagePath = uri;
+                    binding.EProfileImage.setImageURI(uri);
+                }
+            });
+
+    /**
+     * If a user hits the delete image button, this method will generate an avatar from the given link based upon the name field
+     * and set the image view to the URI
+     * @author Jennifer
+     * @param name the name the user has entered
+     * @return void
+     * @version 1.0
+     */
+    public void getAvatar (String name) throws URISyntaxException {
```

Comment:
This isn't the most descriptive name for this method. It isn't clear that this method generates an avatar based on a name just based on the name of this method.